src/Mod/Fem/femcommands/commands.py

```diff
@@ -210,7 +210,7 @@ def __init__(self):
         super(_ConstraintCentrif, self).__init__()
         self.menutext = Qt.QT_TRANSLATE_NOOP(
             "FEM_ConstraintCentrif",
-            "Centrifigual load"
+            "centrifugal load"
```

Comment:
No capital 'C' here ?

Reply:
damn

Reply:
How do I correct it now?

Reply:
Add the change
Then `git rebase -i  HEAD~2`
`fixup` the latest change
`git commit --amend -m "Removing unnecessary strings from Crowdin and improving typos" -m "Patches for https://github.com/FreeCAD/FreeCAD/pull/10459 and https://github.com/FreeCAD/FreeCAD/pull/10426"`
`git push --force`